relay-event-schema/src/protocol/ourlog.rs

```diff
@@ -46,11 +37,54 @@ pub struct OurLog {
     #[metastructure(pii = "true", trim = false)]
     pub attributes: Annotated<Object<AttributeValue>>,
```

Comment:
Do we want to keep this Otel Like format? Could just be a self describing mapping?

```json
            "attributes": {
                "boolean.attribute": {
                    "bool_value": true
                },
```

Could just be:

```json
            "attributes": {
                "boolean.attribute": true,
```

Reply:
I asked for sdk folks to discuss in the develop PR, so we'll see.

Reply:
https://github.com/getsentry/sentry-docs/pull/12920/files#r2012403560